mockData/mockData.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
```

Comment:
I think we should also have an id & title for each bill to properly handle the case for selecting one from multiple past bills and displaying a name for the bill
![image](https://user-images.githubusercontent.com/76412373/227749992-3880f8a9-ea9d-4541-8738-ca932f1bea8d.png)
![image](https://user-images.githubusercontent.com/76412373/227750008-5673a3c0-d6d4-458f-9134-f5f316f0a79b.png)

Reply:
Sounds good, I can input numbers as IDs. As for title of the bill, I think it would make most sense to use the vendor name as the title since that information is already included in a standard receipt. Ex: "Safeway grocery bill" 
How does that sound? @Cosze

Reply:
I think that's fine as a default name property, please be aware that this "name" can be edited by the user though so I don't think we should name it "vendorName". Rather it should be something generic like "name"

Reply:
@Cosze Updated